various-receipes/message-queue/test/unit/mq-example-with-stub.test.js

```diff
@@ -0,0 +1,95 @@
+const request = require("supertest");
+const sinon = require("sinon");
+const { initializeWebServer, stopWebServer } = require("../../../../example-application/api-under-test");
+const { PubSub, Subscription } = require("@google-cloud/pubsub");
+
+let expressApp;
+let sinonSandbox;
+
+beforeAll(async (done) => {
+    // ️️️✅ Best Practice: Place the backend under test within the same process
+    expressApp = await initializeWebServer();
```

Comment:
I suggest that a simple MQ test example should not include an API. A good example is one that removes all the unnecessary details and highlights the important thing.

Reply:
Got it 👍